tls-etcd.tf

```diff
@@ -104,8 +104,8 @@ resource "tls_cert_request" "client" {
     var.etcd_servers,
     list(
       "localhost",
-      "*.kube-etcd.kube-system.svc.cluster.local",
-      "kube-etcd-client.kube-system.svc.cluster.local",
+      "*.kube-etcd.kube-system.svc.${var.cluster_dns_fqdn}",
```

Comment:
With self-hosted etcd gone, we should be able to delete these extra DNS SANs in this file entirely.